### ISSUE TITLE
Install fails on Windows 8

### ISSUE THREAD

**Issue by stephan-nordnes-eriksen** [brunch/sass-brunch#69]
When trying to install the newest sass-brunch it appears that I get an error. The error seems to be with the node-sass package. So I tried to install that manually, and it seems that the newest version of node-sass installs fine (version 2.0.1).

How about upgrading this package for better support on the windows platform? Sorry I don't have a PR, as I do not feed competent enough in node (yet) to actually push changes.

```
npm install --save sass-brunch
npm WARN package.json karma-chrome-launcher@0.1.7 No README data
\
> node-sass@1.2.3 install C:\Users\ME\Documents\Code\project_name\node_modules\sass-brunch\node_modules\node-sass
> node scripts/install.js

Binary downloaded and installed at C:\Users\ME\Documents\Code\project_name\node_modules\sass-brunch\node_modules\node-sass\vendor\win32-x64\binding.node

> node-sass@1.2.3 postinstall C:\Users\ME\Documents\Code\project_name\node_modules\sass-brunch\node_modules\node-sass
> node scripts/build.js

`win32-x64` exists; testing
module.js:355
  Module._extensions[extension](this, filename);
                               ^
Error: The specified procedure could not be found.
C:\Users\ME\Documents\Code\project_name\node_modules\sass-brunch\node_modules\node-sass\vendor\win32-x64\binding.node
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (C:\Users\ME\Documents\Code\project_name\node_modules\sass-brunch\node_modules\node-sass\lib\index.js:181:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "sass-brunch"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! node-sass@1.2.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@1.2.3 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\ME\Documents\Code\project_name\npm-debug.log
```

**Comment by stephan-nordnes-eriksen** [comment]
Just closed? Really? Is this fixed? Is it not relevant to this repo? What is up?

**Comment by es128** [comment]
You asked for the ability to upgrade to node-sass 2, and I just merged a PR that does that.

**Comment by stephan-nordnes-eriksen** [comment]
Alright :) I did not see that.

**Comment by stephan-nordnes-eriksen** [comment]
Could you maybe push this fix in a `1.8.9` release? Would be nice to not use the github repo in my `package.json` :)

**Comment by es128** [comment]
yep, was getting around to that :)
